core/src/main/java/org/apache/carbondata/core/datastore/TableSpec.java

```diff
@@ -280,12 +288,16 @@ public void readFields(DataInput in) throws IOException {
     }
   }
 
-  public class MeasureSpec extends ColumnSpec implements Writable {
+  public static class MeasureSpec extends ColumnSpec implements Writable {
 
     MeasureSpec(String fieldName, DataType dataType) {
       super(fieldName, dataType, ColumnType.MEASURE);
     }
 
+    public static MeasureSpec newInstance(String fieldName, DataType dataType) {
```

Comment:
can we keep  columnName similar to DimensionSpec  , basically make it consistent. if fieldName is columnName then better we can use columnName everywhere as its more readable.  whats your suggestion?